spec/seoshop-api/client_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+module Seoshop
+  RSpec.describe Client do
+
+    subject { described_class.new('123', 'abc', 'http://localhost/seoshop-api') }
+
+    it 'assigns on creation' do
+      expect(subject).to have_attributes(access_token: '123', shop_language: 'abc')
+    end
+
+    context 'instance_methods' do
```

Comment:
extra context here, usually "#method_name" stands for instance notation, ".class_method" correspondently . (c) Ruby docs style.